RedditSharp/Reddit.cs

```diff
@@ -493,7 +489,7 @@ public Listing<Subreddit> SearchSubreddits(string query)
 
         #region Helpers
 
-        protected async internal Task<T> GetThingAsync<T>(string url) where T : Thing
+        protected async internal Task<T> GetThing<T>(string url) where T : Thing
```

Comment:
Might just be the compare, but this shouldn't get renamed as it is an async method